examples/Example.SmartContract.SampleRoyaltyNEP11Token/SampleRoyaltyNEP11Token.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// SampleRoyaltyNEP11Token.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Attributes;
+using Neo.SmartContract.Framework.Native;
+using Neo.SmartContract.Framework.Services;
+using System;
+using System.ComponentModel;
+using System.Numerics;
+
+namespace NonDivisibleNEP11
+{
+    /// <inheritdoc />
+    [DisplayName("SampleRoyaltyNEP11Token")]
+    [ContractAuthor("core-dev", "dev@neo.org")]
+    [ContractVersion("0.0.1")]
+    [ContractDescription("A sample of NEP-11 Royalty Feature")]
+    [ContractSourceCode("https://github.com/neo-project/neo-devpack-dotnet/tree/master/examples/")]
+    [ContractPermission(Permission.WildCard, Method.WildCard)]
+    [SupportedStandards(NepStandard.Nep11)]
+    public class SampleRoyaltyNEP11Token : Nep11Token<Nep11TokenState>
+    {
+        #region Owner
+
+        private const byte PrefixOwner = 0xff;
+
+        private static readonly UInt160 InitialOwner = "NUuJw4C4XJFzxAvSZnFTfsNoWZytmQKXQP";
+
+        [Safe]
+        public static UInt160 GetOwner()
+        {
+            var currentOwner = Storage.Get(new[] { PrefixOwner });
+
+            if (currentOwner == null)
+                return InitialOwner;
+
+            return (UInt160)currentOwner;
+        }
+
+        private static bool IsOwner() => Runtime.CheckWitness(GetOwner());
+
+        public delegate void OnSetOwnerDelegate(UInt160 newOwner);
+
+        [DisplayName("SetOwner")]
+        public static event OnSetOwnerDelegate OnSetOwner;
+
+        public static void SetOwner(UInt160? newOwner)
+        {
+            if (IsOwner() == false)
+                throw new InvalidOperationException("No Authorization!");
+            if (newOwner != null && newOwner.IsValid && !newOwner.IsZero)
```

Comment:
```suggestion
            if (newOwner != null && newOwner.IsValid && newOwner.IsZero == false)
```

Better to write out `if` statements longer. Easier to read and user friendly.